### PR TITLE
ADDED: Remaining Cards and Round Image

### PR DESCRIPTION
**Pull request by munjalSaksham71**
**Added Cards:**
1. Cards with badges  
2. Cards with dismiss 
3. Card with shadow

**Responsive Round Image**

**Corrected Typo of Outline button to icon button**

### CODE REVIEW

File: utils/badge.css [hunk 1]
```diff
@@ -36,9 +36,9 @@
 .badge_btn_left {
     background: var(--secondary);
     color: var(--white);
-    padding: 5px;
+    padding: 8px;
     font-size: 15px;
```
Comment (Nagashree01) [inline utils/badge.css]: you can use rem here

File: docs/docs.html [hunk 2]
```diff
@@ -306,10 +306,97 @@ <h1 class="heading">REGAL UI</h1>
...
+            class="code-snippet"
+            sandbox="allow-scripts allow-same-origin">
+            </iframe> <hr class="mb-3 mt-3" />
+
              <!-- **************** Image Responsive ************************ -->
              <div id="img_res" class="heading1 bolder mt-5 mb-5">Responsive Image</div>
              <div class="overview mb-3">To make image responsive, just use class img-res</div>
-             <img src="../images/harvey.jpeg" class="img-res m-auto"></img>
+             <img src="../images/Image.png" class="img-res m-auto"></img>
```
Comment (Nagashree01) [inline docs/docs.html]: you can specify a alt attribute here

**Review by Nagashree01 — APPROVED** [review]
The UI looks amazing. and code is  well written. Just added one comment.  Good to merge.

File: utils/badge.css [hunk 3]
```diff
@@ -27,7 +27,7 @@
 .badge_btn_right {
     background: var(--secondary);
     color: var(--white);
-    padding: 5px;
+    padding: 8px;
     font-size: 15px;
```
Comment (cjinc121) [inline utils/badge.css]: its better to add values in rem if values are greater than 10-12 px

**Review by cjinc121 — APPROVED** [review]

File: docs/docs.html (hunk 2 above)
Comment (munjalSaksham71) [reply docs/docs.html]: Ok sure

**Pull request merged**